cmd/app/app.go

```diff
@@ -52,6 +54,13 @@ func NewCommand(ctx context.Context) *cobra.Command {
 				return err
 			}
 
+			if errs := validation.ValidateAnnotations(opts.CertManager.AdditionalAnnotations, nil); len(errs) != 0 {
```

Comment:
If we do `field.NewPath("certificate-request-additional-annotations")` rather than nil, does it produce nicer output?

Reply:
```
Error: invalid annotations: certificate-request-additional-annotations: Invalid value: "_foo.baz": name part must consist of alphanumeric characters, '-'
, '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyName',  or 'my.name',  or '123-abc', regex used for validation is '([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]')
```

I reckon we just return `errs.ToAggregate()` (no `fmt.Errorf`), so that we get:
```
Error: certificate-request-additional-annotations: Invalid value: "_foo.baz": name part must consist of alphanumeric characters, '-', '_' or '.', and mus
t start and end with an alphanumeric character (e.g. 'MyName',  or 'my.name',  or '123-abc', regex used for validation is '([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]')
```